packages/replay/package.json

```diff
@@ -51,14 +51,14 @@
     "@types/pako": "^2.0.0",
     "jsdom-worker": "^0.2.1",
     "pako": "^2.0.4",
+    "rrweb": "1.1.3",
     "tslib": "^1.9.3"
   },
   "dependencies": {
     "@sentry/core": "7.22.0",
     "@sentry/types": "7.22.0",
     "@sentry/utils": "7.22.0",
-    "lodash.debounce": "^4.0.8",
-    "rrweb": "^1.1.3"
+    "lodash.debounce": "^4.0.8"
```

Comment:
We can probably think about also either inlining this, or actually replace it with some custom code. but we can tackle this later, I'd say!

Reply:
We have custom (kinda-)debounce logic in [`IdleTransaction`](https://github.com/getsentry/sentry-javascript/blob/450d16d94b5e65d14f1c2bd21a95da5756245f6d/packages/tracing/src/idletransaction.ts#L56) and our [Vue SDK (component tracking)](https://github.com/getsentry/sentry-javascript/blob/450d16d94b5e65d14f1c2bd21a95da5756245f6d/packages/vue/src/tracing.ts#L37-L49). So generally I'm +1 for replacing it with custom code. But as you said, that's definitely something to do for later.